pyvista/plotting/composite_mapper.py

```diff
@@ -0,0 +1,1032 @@
+        ``[d, e, f]``, and ``[g, h, i]``, respectively, then the head node,
+        ``A``, would be the zero index, followed by the first child, ``B``,
+        followed by all the children of ``B``, ``[d, e, f]``. In data
+        structures, this flat indexing would be known as "Depth-first search"
+        and the entire indexing would be::
+
+           ``[A, B, d, e, f, C, g, h, i]``
+
+        Examples
+        --------
+        Add a sphere and a cube as a multiblock dataset to a plotter and then
+        change the visibility and color of the blocks.
+
+        >>> import pyvista as pv
+        >>> dataset = pv.MultiBlock([pv.Cube(), pv.Sphere(center=(0, 0, 1))])
+        >>> pl = pv.Plotter()
+        >>> actor, mapper = pl.add_composite(dataset)
+        >>> mapper.block_attr[1].color = 'b'
+        >>> mapper.block_attr[1].opacity = 0.1
+        >>> mapper.block_attr[1]
+        Composite Block Addr=... Attributes
+        Visible:   None
+        Opacity:   0.1
+        Color:     Color(name='blue', hex='#0000ffff')
+        Pickable   None
+
+        """
+        return self._attr
+
+    @property
+    def color_missing_with_nan(self) -> bool:
+        """Color missing arrays with the NaN color.
+
+        Examples
+        --------
+        Enable coloring missing values with NaN.
+
+        >>> import pyvista as pv
+        >>> dataset = pv.MultiBlock([pv.Cube(), pv.Sphere(center=(0, 0, 1))])
+        >>> pl = pv.Plotter()
+        >>> actor, mapper = pl.add_composite(dataset)
+        >>> mapper.color_missing_with_nan = True
+        >>> mapper.color_missing_with_nan
+        True
+
+        """
+        return self.GetColorMissingArraysWithNanColor()
+
+    @color_missing_with_nan.setter
+    def color_missing_with_nan(self, value: bool):
+        self.SetColorMissingArraysWithNanColor(value)
+
+    @property
+    def lookup_table(self):
+        """Return or set the lookup table.
+
+        Examples
+        --------
+        Return the lookup table.
+
+        >>> import pyvista as pv
+        >>> dataset = pv.MultiBlock([pv.Cube(), pv.Sphere(center=(0, 0, 1))])
+        >>> pl = pv.Plotter()
+        >>> actor, mapper = pl.add_composite(dataset)
+        >>> mapper.lookup_table  # doctest:+SKIP
+        <vtkmodules.vtkCommonCore.vtkLookupTable(0x2d4c6e0) at 0x7fce74a89fa0>
+
+        """
+        return self.GetLookupTable()
+
+    @lookup_table.setter
+    def lookup_table(self, table):
+        return self.SetLookupTable(table)
+
+    @property
+    def scalar_range(self):
+        """Return or set the scalar range.
+
+        Examples
+        --------
+        Return the scalar range. In this example it's set to its default value
+        of ``(0.0, 1.0)``.
+
+        >>> import pyvista as pv
+        >>> dataset = pv.MultiBlock([pv.Cube(), pv.Sphere(center=(0, 0, 1))])
+        >>> pl = pv.Plotter()
+        >>> actor, mapper = pl.add_composite(dataset)
+        >>> mapper.scalar_range
+        (0.0, 1.0)
```

Comment:
Another blank example.

Reply:
Fixed in 8835c292f03a212cc953200b58f421e82e67c120.